### ISSUE TITLE
Missing textures

### ISSUE THREAD

**Issue by LeXillal** [WiIIiam278/WebSchematics#2]
Since it comes from a git that contains all the textures it should work for all textures, right?
![image](https://user-images.githubusercontent.com/14876290/213944131-9e2a8423-3e78-411c-b996-24f972d4cca1.png)

**Comment by WiIIiam278** [comment]
Yes, but my logic makes an assumption that blocks map to just one model json, when there's a bunch of blocks with special cases, like glass panes and fences
![image](https://user-images.githubusercontent.com/31187453/213945560-13b87cda-06de-4fd9-93f7-b32f2308f20f.png)
so, we'd need to apply special models to blocks based on their blockstate data if we wanted these to render correctly
